packages/bar/src/Bar.tsx

```diff
@@ -368,6 +370,11 @@ const InnerBar = <RawDatum extends BarDatum>({
         )
     }
 
+    if (layers.includes('totals') && enableTotals) {
+        const barTotals = computeBarTotals(bars, xScale, yScale, layout, groupMode, totalsOffset)
```

Comment:
Ideally, this should be memoized, in a hook, there's a bit of logic involved so it might be best to not have to recompute it all the time, as even hovering a bar would lead to recompute this.

Reply:
I did it and passed the computation to `useBar` hook

Reply:
Perfect, thank you.